classes/controller/FrontController.php

```diff
@@ -1064,9 +1064,13 @@ public function registerStylesheet($id, $relativePath, $params = array())
             'inline' => false,
             'server' => 'local',
         ];
-
         $params = array_merge($default_params, $params);
 
+        if (Tools::hasMediaServer() && !Configuration::get('PS_CSS_THEME_CACHE')) {
+            $relativePath = Tools::getCurrentUrlProtocolPrefix() . Tools::getMediaServer($relativePath)
```

Comment:
Not sure 
```php
Tools::getCurrentUrlProtocolPrefix() . Tools::getMediaServer($relativePath)
                . $this->stylesheetManager->getFullPath($relativePath)
``` 
can be empty / null :thinking:

Reply:
@PierreRambaud It's not this part but `$this->stylesheetManager->getFullPath($relativePath)` which can be equals to false